feature: multiple standards support ALA-1140 (#2179)
codacy/docs: docs/organizations/using-coding-standards.md

    Codacy will start using the new coding standard on the next analysis of each selected repository.

    !!! note

>         A repository can follow multiple coding standards at a time.

Reviewer: We can remove this note entirely now 👍

1.  Click the trash can icon on the coding standard card and confirm.

    ![Deleting a coding standard](images/coding-standard-delete.png)

## Using a coding standard alongside tool configuration files {: id="using-with-tool-configuration"}

When using a [tool configuration file](../repositories-configure/configuring-code-patterns.md#using-your-own-tool-configuration-files) alongside a coding standard, the configuration file controls the code patterns, while the coding standard controls whether the tool is enabled or disabled.

## Using multiple coding standards {: id="using-multiple"}

When Codacy analyzes your code using multiple coding standards, it merges the tools and patterns from each standard. Enabled tools and patterns in any applied standard will be enforced.

!!! warning

>     If the same pattern appears in multiple standards, the version from the most recently applied standard (with its parameters) will be prioritized.

Reviewer: I suggest using a stronger verb than "prioritize", as it implies that the "lower priority" patterns still have some effect. Also, the best practice with docs is to use the present tense.

```suggestion
    If the same pattern appears in multiple standards, the version from the most recently applied standard (with its parameters) takes precedence.
```
